### PR TITLE
Add styled paragraph underline usage example

### PR DESCRIPTION
**Pull request by adrg**

### CODE REVIEW

File: report/README.md [hunk 1]
```diff
@@ -6,5 +6,4 @@ The example explains how to create PDF reports using UniPDF, it also shows you h
 
 - [pdf_report.go](pdf_report.go) The example showcases PDF report generation with UniPDF's creator package. The output is saved as unidoc-report.pdf which illustrates some of the features of the creator.
 - [pdf_tables.go](pdf_tables.go) The example showcases PDF tables features using UniPDF's creator package. The output is saved as UniPDF-tables.pdf which illustrates some of the features of the creator.
-- [pdf_formatted_text.go](styled_par.go) The example showcases the creation of styled paragraph. The output is saved as styled_paragraph.pdf which illustrates some of the features of the creator.
--[pdf_creator_font_subset.go](pdf_creator_font_subset.go) The example showcases how to subset fonts embedded in the PDF file to reduce its size.
```
Comment (gunnsth) [inline report/README.md:10]: is this example removed?

Comment (adrg) [reply report/README.md:10]: The example has been moved into the `compress` directory at some point. Not really sure when that happened. I noticed the README file was referencing a broken example link, so I removed that line.

Comment (gunnsth) [reply report/README.md:10]: ok

**Review by gunnsth — APPROVED** [review]
LGTM

**Pull request merged**